src/native/managed/cdacreader/tests/CodeVersionsTests.cs

```diff
@@ -63,202 +64,80 @@ public static MockMethodDesc CreateVersionable (TargetPointer selfAddress, Targe
             };
         }
     }
+
     internal class MockCodeBlockStart
     {
         public TargetCodePointer StartAddress { get; set;}
         public uint Length { get; set; }
-
-        public bool Contains(TargetPointer ip) =>  ip >= StartAddress && ip < StartAddress + Length;
-        public bool Contains(TargetCodePointer ip) => Contains(ip.AsTargetPointer);
         public MockMethodDesc MethodDesc {get; set;}
-        public TargetPointer MethodDescAddress => MethodDesc.Address;
     }
 
-    internal class MockExecutionManager : IExecutionManager
-    {
-        private IReadOnlyCollection<MockCodeBlockStart> _codeBlocks;
+    private Mock<IRuntimeTypeSystem> mockRuntimeTypeSystem = new(MockBehavior.Strict);
+    private Mock<IExecutionManager> mockExecutionManager = new(MockBehavior.Strict);
+    private Mock<ILoader> mockLoader = new(MockBehavior.Strict);
```

Comment:
`MockBehavior.Strict` isn't required, but I wanted to make sure I was capturing all the calls.